contents/docs/user-guides/paths.mdx

```diff
@@ -6,26 +6,192 @@ showTitle: true
+
+Start and end points can be set to a number of different options:
+
+-   Pageview URLs - Only available when [Page views](#page-views) are set as an event type, and each option is a unique URL
+-   Screens - Only available when [Screen views](#screen-views) are set as an event type, and each option is a unique name corresponding to the `$screen_name` property
+-   Custom events - Only available when [Custom events](#custom-events) are set as an event type. Options are all events other than `$pageview` and `$screen`
+-   Wildcard groups - If you've created a [Wildcard group](#wildcard-groups), you can set that as either the start or end point as well
+
+### Exclusions
+
+Exclusions allow you to remove events entirely from being displayed in a path, and can be added for Pageview URLs, Screens, Custom events, and Wildcards.
+
+![excluding two events](../../images/docs/user-guides/paths/exclusions.png)
+
+### Advanced options
+
+Paths provide a number of more advanced configuration options for customizing the behavior of the insight.
+
+#### Maximum number of paths
+
+This is the maximum number of paths that will be displayed. Here, a 'Path' refers to a movement of a user from one step to another, and does not correspond to the number of paths from start to finish.
+The default number is 50, but oftentimes this will be far too cluttered and reducing the number will make it much easier to analyze.
+
+#### Number of people on each path
+
+This option allows you to set two numbers corresponding to the minimum and maximum number of users that can exist on a path. Once again, in this case 'Path' is referring to a movement from one step to another, not from start to finish.
+If the number of users who moved from one step to another is either lower than the minimum, or higher then the maximum, this still will not be displayed. If one of the options is set blank, this means there is no limit for that specific bound.
+
+This option is very useful when analyzing paths that are visually dense, and allows you to filter off paths with very few users on them to get a better idea of what the majority of users are doing.
+
+#### Path cleaning rules
 
-Paths work across subdomains, even with the basic snippet installation. For example, it's common for website.com to be the landing page, then traffic to move into app.website.com once a user starts interacting with the product. Just set up the root domain as the URL in your [tracking settings](/docs/integrate/client/snippet-installation).
+Path cleaning rules are an advanced feature that can be used to standardize the form of URLs within a Paths visualization. These rules consist of two parts: a regex which is used to match the various URLs you would like to combine, and an alias, which is the name given to any events that matched the regex.
+In this sense, these path cleaning rules are an advanced for of [Wildcard groups](#wildcard-groups), which use regex to match instead of the simple `*` wildcard.
```

Comment:
One regex example here would be excellent, like how to clean URLs of the form: `api/projects/<numeric_id>/persons/<person_id>/`